rtrlib/rtr_mgr.c

```diff
@@ -377,37 +431,77 @@ int rtr_mgr_init(struct rtr_mgr_config **config_out,
 	return err_code;
 }
 
+static struct rtr_mgr_group *rtr_mgr_get_first_group(struct rtr_mgr_config
```

Comment:
This might also be changed in a separate PR that does some refactoring. In my humble opinion, I think the name is OK, the _get_ prevents confusion over what exactly the function does, I think.